tests/unit/test_torch_dataloader.py

```diff
@@ -366,7 +366,6 @@ def test_mh_support(tmpdir):
     assert idx > 0
 
 
-@pytest.mark.skip(reason="fails with cuda illegal memory access")
```

Comment:
nice - glad you figured out what was going on with this test (though in retrospect we probably should not have skipped this )